Alloy/lib/alloy/controllers/BaseController.js

```diff
@@ -154,12 +218,39 @@ var Controller = function() {
 		 * @method createStyle
 		 * Creates a dictionary of properties based on the specified styles.
 		 *
+		 *
 		 * You can use this dictionary with the view object's
 		 * {@link Titanium.UI.View#method-applyProperties applyProperties} method
 		 * or a create object method, such as {@link Titanium.UI#method-createView Titanium.UI.createView}.
+		 * #### Examples
+		 * The following creates a new style object that passed as a parameter 
+		 * to the {@link Titanium.UI#method-createLabel Ti.UI.createLabel()} method.
+
+	var styleArgs = {
+	apiName: 'Ti.UI.Label',
+		classes: ['blue','shadow','large'],
+		id: 'tester',
+		borderWidth: 2,
+		borderRadius: 16,
+		borderColor: '#000'
+	};
+	var styleObject = $.createStyle(styleArgs);
+	testLabel = Ti.UI.createLabel(styleObject);	 
+
+		 * The next example uses the {@link Titanium#method-applyProperties applyProperties()} method
+		 to add apply a style object to an existing Button control (button not shown).
```

Comment:
This line is being code formatted.  Probably need an asterisk at the beginning of the line.